src/components/Container.js

```diff
@@ -50,8 +50,10 @@ const StyledContent = styled(motion.div)`
 `
 
 const InnerContent = styled.div`
+  display: flex;
   flex: 1 1 auto;
-  padding-bottom: ${spacing.padding.small}px;
+  ${props =>
```

Comment:
@kevinrademan I'm concerned about the display and padding change. Have you check to see that it all still looks correct in the other iterations? I feel like there was a reason this changed..., maybe the branding stuff, or the median?

Reply:
I've gone through the kb, and contact and I couldnt find anything that looks out of place. To my knowledge the only place where this particular padded prop is getting used is actually in the chat system. All other parts of the app have it enabled